include/bgfx/c99/bgfx.h

```diff
@@ -145,11 +145,13 @@ typedef enum bgfx_attrib
  */
 typedef enum bgfx_attrib_type
 {
-    BGFX_ATTRIB_TYPE_UINT8,                   /** ( 0) Uint8                          */
-    BGFX_ATTRIB_TYPE_UINT10,                  /** ( 1) Uint10, availability depends on: `BGFX_CAPS_VERTEX_ATTRIB_UINT10`. */
-    BGFX_ATTRIB_TYPE_INT16,                   /** ( 2) Int16                          */
-    BGFX_ATTRIB_TYPE_HALF,                    /** ( 3) Half, availability depends on: `BGFX_CAPS_VERTEX_ATTRIB_HALF`. */
-    BGFX_ATTRIB_TYPE_FLOAT,                   /** ( 4) Float                          */
+    BGFX_ATTRIB_TYPE_INT8,                    /** ( 0) Int8, availability depends on: `BGFX_CAPS_VERTEX_ATTRIB_INT8`. */
```

Comment:
Are we 100% sure this enum is not serialized anywhere?

Reply:
Would it be safer to add the new enums to the end?

Reply:
Assignment of IDs happens in vertexlayout.cpp, see s_attribTypeToId

Reply:
Yes, I see that. I'm just not sure this is used directly in bgfx's shader compiler or something.